build

```diff
@@ -19,4 +19,4 @@
 # under the License.
 #
 
-mvn clean install -Dmaven.test.skip=true
+mvn clean install -Dmaven.test.skip.exec=true
```

Comment:
maven.test.skip.exec means compile and not executed, while previously it does not even compile. That is my understanding. 

What is the issue previously?

Reply:
I see. Previously admin may compiles failing but no one notice.

Reply:
I removed compiling before to improve the build time, because readme also doesn't compile the tests. Then today I was aware that helix admin webapp depends on helix-core test..

If you remove the local mvn cached jars and build, it would fail.